#997: Added documentation for Rollback of Script-Languages-Container (#999)
exasol/script-languages-release: doc/user_guide/use_R3_and_Python2_on_7.1.30_and_above.md

# Use R3.6 UDF's with Exasol 7.1.30 and later

## Background

> The Script-Languages-Container was updated to version [8.1.0](https://github.com/exasol/script-languages-release/releases/tag/8.1.0) in Database v7.1.30. Release 8.1.0 included a major update for the Python and R languages compared to 6.0.0. For R we detected some performance drops in special circumstances: Newer versions of R have a higher memory consumption which has direct impact on the execution of UDF's in parallel. 

Reviewer: ```suggestion
The Script-Languages-Container was updated to version [8.1.0](https://github.com/exasol/script-languages-release/releases/tag/8.1.0) in Database v7.1.30. Release 8.1.0 included a major update for the Python and R languages compared to 6.0.0. For R we detected some performance drops in special circumstances: Newer versions of R have a higher memory consumption which can lead to throttling of the UDFs and decrease the their performance. 
```

Author: Done

Also, some R packages were removed. 

> In case, customers want to use the previous Script-Languages-Container [6.0.0](https://github.com/exasol/script-languages-release/releases/tag/6.0.0), they can follow the steps described in this document.

Reviewer: We should mention, that limiting the instance count can also avoid the throttling for the cost of some parallelism.
